README.md

```diff
@@ -104,28 +104,21 @@ from pyfcm import FCMNotification
 
 from fcm_devices.models import Device
 from fcm_devices.settings import app_settings
-
-push_service = FCMNotification(api_key=app_settings.API_KEY)
-data_message = {
-    "Nick": "Mario",
+from google.oauth2 import service_account
+push_service = FCMNotification(credentials=service_account.Credentials.from_service_account_info(app_settings.GOOGLE_SERVICE_ACCOUNT_INFO))
+data_payload = {
+    "foo": "bar",
     "body": "great match!",
     "Room": "PortugalVSDenmark"
 }
 
 device = Device.objects.get(user_id=123)  # get the device for the user you want to message
 
-# To multiple devices
```

Comment:
note that the [new API no longer supports multicasting](https://firebase.google.com/support/faq?authuser=0&_gl=1*ldgs3m*_up*MQ..*_ga*ODYyNzMzOTQ5LjE3MjQyNTcxMjA.*_ga_CW55HF8NVT*MTcyNDI1NzExOS4xLjAuMTcyNDI1NzExOS4wLjAuMA..#fcm-depr-multiple-tokens)